src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/StringUtils.java

```diff
@@ -121,6 +121,15 @@ public static boolean isQuoted(String text, String quote) {
         return !Strings.isNullOrEmpty(text) && text.startsWith(quote) && text.endsWith(quote);
     }
 
+    public static boolean isNumeric(String text) {
```

Comment:
we're using Guava, so this can be replaced by 

Doubles.tryParse(text) != null;

Reply:
Great, I'll try this out. Thanks!